src/app/utils/question.ts

```diff
@@ -0,0 +1,63 @@
+import { db } from "../../../firebase";
```

Comment:
nit: `question.ts` and `course.ts` can live in `services` maybe under a folder called `client.ts`?